packages/core/src/generators/builder/generator.ts

```diff
@@ -324,17 +324,42 @@ type InternalOptions = {
   skipMapper?: boolean;
 };
 
+const processLocalizedValues = (element: BuilderElement, node: MitosisNode) => {
+  const propertiesPrefix = 'properties.';
+  const componentOptionsPrefix = 'component.options.';
+
+  if (node.localizedValues) {
+    for (const [path, value] of Object.entries(node.localizedValues)) {
+      if (path.startsWith(propertiesPrefix)) {
+        const key = path.replace(propertiesPrefix, '');
+        element.properties = element.properties || {};
+        (element as any).properties[key] = value;
+      } else if (path === 'linkUrl') {
+        (element as any).linkUrl = value;
+      } else if (path.startsWith(componentOptionsPrefix)) {
+        const key = path.replace(componentOptionsPrefix, '');
+        if (element.component) {
+          element.component.options[key] = value;
+        }
+      }
```

Comment:
you can probably use a `set` helper to do this (I suggest copy/pasting https://github.com/BuilderIO/builder/blob/main/packages/sdks/src/functions/set.ts instead of using the `lodash` version to keep bundle size low)

```suggestion
      set(element, path, value)
```

it would be more future proof: you won't need to add more cases here as time goes on